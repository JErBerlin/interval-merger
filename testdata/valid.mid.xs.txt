[150,170] [200,220] [300,320] [210,230]
