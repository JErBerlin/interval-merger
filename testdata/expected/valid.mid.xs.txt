[150,170] [200,230] [300,320] 
